README.md

```diff
@@ -399,7 +400,7 @@ except KeyError, exc:
   honeybadger.notify(exc, context={'foo': 'bar'})
 
 # with custom arguments
-honeybadger.notify(error_class='ValueError', error_message='Something bad happened!')
+honeybadger.notify(error_class='ValueError', error_message='Something bad happened!', context={'fingerprint': 'custom_fingerprint'})
```

Comment:
Is there any way we could do this instead? This is more in line with how the other client libraries work. Fingerprint is a top-level option.

```suggestion
honeybadger.notify(error_class='ValueError', error_message='Something bad happened!', fingerprint='custom_fingerprint')
```

Reply:
> It's doable, I'm thinking whether this would be a breaking change 🤔
> However, can you point me to the other libraries? Because, I actually looked [here](https://github.com/honeybadger-io/honeybadger-ruby/blob/master/lib/honeybadger/agent.rb#L107) and at the js libraries and fingerprint is not a top-level option.

I think I'm misunderstanding. In the Ruby code you linked to, `fingerprint` is a top-level option like `context` is. So you can do this:

```
Honeybadger.notify(error, fingerprint: 'foo')
```

or this:

```
Honeybadger.notify(error, context: {key: 'value'})
```

or this:

```
Honeybadger.notify(error,
  context: {key: 'value'},
  fingerprint: 'foo'
)
```

It doesn't get the fingerprint from the context. Note the context in Honeybadger is a special data structure that is meant for extra data provided by the user.

Reply:
Ah yes sorry, I misread the code. I will create a new PR now.